templates/corporate/billing.html

```diff
@@ -69,14 +69,25 @@ <h1>{{ _("Billing") }}</h1>
                     <div class="tab-pane" id="loading">
                     </div>
                 </div>
-
                 <div class="support-link">
                     <p>
                         Contact <a href="mailto:support@zulipchat.com">support@zulipchat.com</a>
                         for billing history or to make changes to your subscription.
                     </p>
                 </div>
-
+                {% elif admin_access and not has_active_plan %}
+                <p>
+                    <div class="tab-content">
+                        <p>
+                            You are not subscribed to any paid plans. Visit <a href="/plans">plans</a> page to upgrade.
```

Comment:
Is that the right way to upgrade from /billing?

Reply:
I think so. I will add some tests in test_stripe(if not already present) to make sure it works.